common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java

```diff
@@ -211,16 +214,18 @@ protected void renewAccessToken(@NonNull final AcquireTokenSilentOperationParame
                     "Token request was successful"
             );
 
-            final ICacheRecord savedRecord = tokenCache.save(
+            // TODO this is the new result that will become the TenantProfile...
+            final List<ICacheRecord> savedRecords = tokenCache.saveAndLoadAggregatedAccountData(
                     strategy,
                     getAuthorizationRequest(strategy, parameters),
                     tokenResult.getTokenResponse()
             );
+            final ICacheRecord savedRecord = savedRecords.get(0);
 
             // Create a new AuthenticationResult to hold the saved record
             final LocalAuthenticationResult authenticationResult = new LocalAuthenticationResult(
                     savedRecord,
-                    parameters.getSdkType()
+                    savedRecords
```

Comment:
What happened to SDK type here?

Reply:
I added [another constructor that specifies its type as MSAL by default](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/iambmelt/tenant-profile/common/src/main/java/com/microsoft/identity/common/internal/result/LocalAuthenticationResult.java#L58-L61).

Reply:
I'll probably remove this additional constructor and pass-in the param, though. In an upcoming commit...

Reply:
Resolved in 70dbc62076909d5d18aa4471540145288ab000a8